.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
 before_install:
 - echo $TRAVIS_OS_NAME
 - export CSL_SYSTEM_TAG=$TRAVIS_OS_NAME
-- export DCONFIG=travis
+- export DCONFIG=qanet_tns_wallet
```

Comment:
This issue and PR are only about making some constants flexible which allows us to get rid of `travis` mode. It's the first step to using new conventions. After it's done we will be able to easily change these names (but in another issue/PR).